src/main/java/seedu/address/model/person/Star.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Student's stars in the address book.
+ */
+public class Star {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Stars given should be more than 0.";
+
+    public static final Star NO_STAR = new Star(0);
+
+    public final Integer numOfStars; // number of stars given to a student
+
+    /**
+     * Constructs a {@code Star}.
+     *
+     * @param numOfStars A valid number.
+     */
+    public Star(Integer numOfStars) {
+        requireNonNull(numOfStars);
+        checkArgument(isValidStar(numOfStars), MESSAGE_CONSTRAINTS);
+        this.numOfStars = numOfStars;
+    }
+
+    /**
+     * Returns true if a given string is a valid number.
+     */
+    public static boolean isValidStar(Integer noOfStars) {
```

Comment:
I think you forgot to change `noOfStars` to `numOfStars` here.

It would adhere closer to the coding standards.

Reply:
Okay.